src/bigquery.ts

```diff
@@ -90,7 +90,6 @@ export type Query = JobRequest<bigquery.IJobConfigurationQuery> & {
   params?: any[] | {[param: string]: any};
   dryRun?: boolean;
   types?: string[] | string[][] | {[type: string]: string[]};
-  defaultDataset?: Dataset;
```

Comment:
Why did the property get deleted from the interface?

Reply:
it's already there in `IJobConfigurationQuery`.
types are different one is `Dataset` and another one is `IDatasetReference`.